README.md

```diff
@@ -60,38 +70,37 @@ echidna_convert: failed!💥
     mint(57896044618658097711785492504343953926634992332820282019728792003956564819968)
 ```
 
+Echidna supports testing complex contracts with two features. First, it [allows initializing a fuzzing campaign with arbitrary transactions using Etheno and Truffle](https://github.com/crytic/echidna/wiki/Deployment-of-a-contract-using-Truffle,-Ganache-and-Etheno-to-test-with-Echidna). Second, it can test more than one contract at the same time, calling any public or external function of any tested contract. Use `multi-abi: true` in your configuration file to allow Echidna to generate transactions to call more than one contract at the same time.
+
 ### Configuration options
 
 Echidna's CLI can be used to choose the contract to test and load a configuration file.
 
 ```
-$ echidna-test contract.sol --contract TEST --config="config.yaml"
+$ echidna-test contract.sol --contract TEST --config config.yaml
 ```
 
 The configuration file allows users to choose EVM and test generation
 parameters. An example of a complete and annotated config file with the default options can be found at [examples/solidity/basic/default.yaml](examples/solidity/basic/default.yaml). More detailed documentation on the configuration options is available in our [wiki](https://github.com/trailofbits/echidna/wiki/Config).
 
 ## Installation
 
-If you want to quickly test Echidna in Linux, we offer a statically linked binary release of v1.5.0.0 to download [here](https://github.com/crytic/echidna/releases/tag/v1.5.0.0).
-
-Otherwise, to install the latest revision of Echidna, we recommend to use [docker](https://www.docker.com/):
+If you want to quickly test Echidna in Linux or MacOS, download a [statically-linked binary release] (https://github.com/crytic/echidna/releases). GitHub Actions also [compiles binaries](https://github.com/crytic/echidna/actions?query=workflow%3ACI+branch%3Amaster+event%3Apush) from every commit to `master`, just click the commit to find binaries for Linux or MacOS.
```

Comment:
we provide statically linked binaries built on Ubuntu and mostly static MacOS binaries built on Mojave. You can also download built artifacts from our actions pipeline or download a pre-built container from the [official github package](https://github.com/crytic/echidna/packages/136575).